exe/awskeyring

```diff
@@ -9,7 +9,7 @@ end
 
 begin
   AwskeyringCommand.start
-rescue Keychain::UserCancelledError => err
-  warn err.to_s
+rescue Keychain::UserCancelledError => e
```

Comment:
I don't agree with the use of single char variable names in interpreted languages. We aren't writing C. I also don't think this is the correct place to merge these changes. I think they should be pulled into a separate PR for clarity of objective.

Reply:
from https://github.com/rubocop-hq/rubocop/pull/6460 the default name is "e", I'm not fussed as to what its called and I am going to go with the default.

Reply:
Fair call, will defer to the wizards of linting here.